src/AddressBookController.ts

```diff
@@ -1,4 +1,4 @@
-import BaseController, { BaseConfig, BaseState } from './BaseController';
+import BaseController, { BaseConfig, BaseState } from '../src/BaseController';
```

Comment:
I think this import should still be `'./BaseController'`.

Reply:
oh yeah good catch thanks